packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -357,7 +357,8 @@ class WebpackCLI {
 
                     try {
                         pkg = await promptInstallation(pkg, () => {
-                            logger.error(`For using this command you need to install: '${green(commandName)}' package`);
+                            const pkgName = pkg.slice(pkg.indexOf('/') + 1);
```

Comment:
```suggestion
                            const pkgName = builtInExternalCommandInfo.name;
```

Reply:
The package associated with `loader`/`plugin` commands is `generators`, which won't be suggested accordingly in this case.

Reply:
Yup, you are right. Then just specify the full package name `@webpack-cli/generators` instead of `generators` which is current output.
```suggestion
```